app-admin/calamares/calamares-3.3.14.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_TEST="true"
+PYTHON_COMPAT=( python3_{10..13} )
+
+QTMIN="6.7.1"
+KFMIN="6.0.0"
```

Comment:
We're already shipping KF 6.10.0 as lowest (stable) version in ::gentoo. You could bump this to `KFMIN=6.9.0` and adapt to the ensuing ecm.eclass behavior change, manually handling (mostly only inheriting, in your case) `xdg.eclass`.

Reply:
Are you sure you don't need xdg.eclass though?

Reply:
> Are you sure you don't need xdg.eclass though?

Already added, both the version and the eclass, in the last push from 1 hour ago :+1: